src/api/airdropApi.ts

```diff
@@ -0,0 +1,39 @@
+import { AirdropCheckWalletsSubscription } from 'app/api';
+import { WalletPayload } from 'app/consts';
+
+import api from './client';
+
```

Comment:
comments in this file left so I don't have to look for/implement different responses when we have API in backend working or want to test different flows. Will remove them before merging to develop. Some day.